nucliadb_vectors/src/relations/bfs_engine.rs

```diff
@@ -173,6 +192,7 @@ mod test {
             .unwrap();
         let expected = &nodes;
         let result = bfs.search().unwrap();
+        let result: Vec<_> = result.map(|cnx| cnx.to()).collect();
```

Comment:
It is more idiomatic to put the `Vec<_>` stuff directly after the method call such as:
```rust
let result = result.map(|cnx| cnx.to()).collect::<Vec<_>>();
```